### ISSUE TITLE
Android: Permission Error in 5.x when Generating Thumbnails

### ISSUE THREAD

**Issue by danbunkr** [dragermrb/capacitor-plugin-video-editor#9]
When trying to use the thumbnail function on Capacitor Android to generate a thumbnail preview I'm getting the following error thrown: "User denied access to storage"

Note: I've also tried adding the following permissions:

uses-permission android:name="android.permission.READ_MEDIA_VIDEO"
Android Version:  ~ Android 13 (API level 33)

Code:

```
        if (uri && size) {
          try {
            const { file } = await VideoEditor.thumbnail({
              path: normalizeVideoPath(uri),
              width: 600,
              height: 600,
              at: 0,
            });

            setFileThumbnail(Capacitor.convertFileSrc(file.path));
          } catch (error) {
            console.log('THUMB ERROR**', error);
            appInsights.trackException({
              exception: error,
              severityLevel: SeverityLevel.Error,
              properties: { name: 'thumbnail creation', fileId },
            });
          }
        }
            
 ```

I have what I believe is the correct permissions in AndroidManifest.xml

```
   <uses-permission android:name="android.permission.INTERNET" />
    <uses-permission android:name="android.permission.CAMERA" />
    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
    <uses-permission android:name="android.permission.USE_BIOMETRIC" />
    <uses-permission android:name="android.permission.READ_CONTACTS" />
    <uses-permission android:name="android.permission.WRITE_CONTACTS"/>
    <uses-permission android:name="com.android.vending.BILLING"/>
    <uses-permission android:name="android.permission.READ_MEDIA_IMAGES"/>
    <uses-permission android:name="android.permission.RECORD_AUDIO" />
 ```

Logcat Info:
```
callback: 63712670, pluginId: VideoEditor, methodName: thumbnail, methodData: {"path":"\/storage\/emulated\/0\/Android\/data\\/files\/2518e2d2-f53e-409e-ae59-3ea78f733038-PXL_20221102_142143045.mp4","width":600,"height":600,"at":0}
2023-07-25 14:16:19.351  8950-8950  Capacitor   D  Sending plugin error: {"save":false,"callbackId":"63712670","pluginId":"VideoEditor","methodName":"thumbnail","success":false,"error":{"message":"User denied access to storage"}}

```

**Comment by dragermrb** [comment]
Hi Dan

I'm on vacation for two weeks, I'll look at it when I get back

**Comment by danbunkr** [comment]
@dragermrb - I know you are vacation and hope you enjoy it. I put up a PR to fix this issue here as it is a blocker for an upcoming release we have. If you wouldn't mind taking a look. Just trying to avoid having to point our app towards a local fork.

https://github.com/dragermrb/capacitor-plugin-video-editor/pull/12

Thanks!

**Comment by dragermrb** [comment]
Merged #12
